upload-application.sh

```diff
@@ -36,13 +36,22 @@ UploadApplication() {
         args+=(--latest)
     fi
 
-    DATADOG_API_KEY="${api_key}" \
+    output=$(DATADOG_API_KEY="${api_key}" \
     DATADOG_APP_KEY="${app_key}" \
     DATADOG_SUBDOMAIN="app" \
     DATADOG_SITE="${site}" \
     DATADOG_SYNTHETICS_CI_TRIGGER_APP="bitrise_step" \
         $DATADOG_CI_COMMAND synthetics upload-application \
-        "${args[@]}"
+        "${args[@]}")
+
+    echo "This is the output: $output"
```

Comment:
When looking at [these docs](https://devcenter.bitrise.io/en/references/steps-reference/step-outputs-reference.html#step-outputs-reference), the examples are in uppercase letters, and [with this GitHub search](https://github.com/search?q=path%3Astep.yml%20%22outputs%3A%22&type=code), I found a [good example of naming](https://github.com/sotayamashita/bitrise-steplib/blob/ac3e254c538090f215c2e2c902d081ada134fa16/steps/autify-upload/0.0.2/step.yml#L61).

I guess we could name our output `DATADOG_UPLOADED_APPLICATION_VERSION_ID`. Not a problem if the name is long, and I like to use the past tense ("uploaded") because it's something which _happened_: it's not an input, but an output.

---

**nit:** was probably only for testing in draft, but no need for "this is the output"

```suggestion
    echo "$output"
```

Reply:
Yeah this was me still trying to figure out if it works as i think it does.
Good suggestion for the name 👍